### PR TITLE
Kafka topic creation

### PR DESCRIPTION
**Pull request by sagikazarmark**
## Overview

Move ingest kafka topic creation from ksqldb

Related #102

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -112,6 +112,17 @@ services:
       KMAGIC_ALLOW_TOPIC_DELETE: "true"
       KMAGIC_ALLOW_SCHEMA_DELETE: "true"
 
+  redpanda:
```
Comment (tothandras) [inline docker-compose.yaml:115]: Should we remove Kafka Magic UI?

File: main.go [hunk 2]
```diff
@@ -169,8 +169,44 @@ func main() {
...
+	})
+	if err != nil {
+		logger.Error("create default Kafka topics: %v", err)
+		os.Exit(1)
+	}
+
+	for _, r := range result {
+		if r.Error.Code() != kafka.ErrNoError {
+			if err != nil {
+				logger.Error("create default Kafka topic %q: %v", r.Topic, err)
```
Comment (tothandras) [inline main.go:200]:
```suggestion
				logger.Error("create default Kafka topic", "topic", r.Topic, "error", err)
```

File: main.go [hunk 3]
```diff
@@ -169,8 +169,44 @@ func main() {
...
+
+	kafkaAdminClient, err := kafka.NewAdminClient(kafkaConfig)
+	if err != nil {
+		logger.Error("init Kafka client: %v", err)
+		os.Exit(1)
+	}
+
+	const eventsTopic = "om_events"
+
+	logger.Info("create default Kafka topics")
```
Comment (tothandras) [inline main.go:182]:
```suggestion
	logger.Debug("create default Kafka topics")
```

**Review by tothandras — CHANGES_REQUESTED** [review]

File: docker-compose.yaml (hunk 1 above)
Comment (sagikazarmark) [reply docker-compose.yaml:115]: Redpanda has more features and open source. KafkaMagic has some basic features behind a paywall (like creating and deleting topics), so KafkaMagic is less useful IMO.

**Review by tothandras — APPROVED** [review]

**Pull request merged**